### PR TITLE
New Data Source: `azurerm_scheduler_job_collection`

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: website/docs/d/scheduler_job_collection.html.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_scheduler_job_collection"
+sidebar_current: "docs-azurerm-datasource-scheduler_job_collection"
+description: |-
+  Get information about the specified schelduer job collection.
```
Comment (tombuildsstuff) [inline website/docs/d/scheduler_job_collection.html.markdown]: scheduler*

File: website/docs/d/scheduler_job_collection.html.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Scheduler Job Collection. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Scheduler Job Collection. Changing this forces a new resource to be created.
```
Comment (tombuildsstuff) [inline website/docs/d/scheduler_job_collection.html.markdown]: We're not creating anything - so we can we change this to `Specifies the name of the Resource Group in which the Scheduler Job Collection exists`

File: website/docs/d/scheduler_job_collection.html.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+output "job_collection_state" {
+  value = "${data.azurerm_scheduler_job_collection.jobs.state}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Scheduler Job Collection. Changing this forces a new resource to be created.
```
Comment (tombuildsstuff) [inline website/docs/d/scheduler_job_collection.html.markdown]: We're not creating anything - so we can we change this to `Specifies the name of the Scheduler Job Collection`

**Comment by katbyte** [comment]
Tests pass:
screen shot 2018-03-20 at 20 33 31

**Pull request merged**